scaleway/resource_lb_backend_beta.go

```diff
@@ -72,8 +72,10 @@ func resourceScalewayLbBackendBeta() *schema.Resource {
 			"server_ips": {
 				Type: schema.TypeList,
 				Elem: &schema.Schema{
-					Type:         schema.TypeString,
-					ValidateFunc: validation.SingleIP(),
+					Type: schema.TypeString,
+					ValidateFunc: func(i interface{}, s string) ([]string, []error) {
+						return validation.IsIPAddress(i, "server_ips")
```

Comment:
same question than above

Reply:
Fixed in https://github.com/scaleway/terraform-provider-scaleway/pull/577